localstack/services/s3/virtual_host.py

```diff
@@ -9,20 +9,19 @@
 from localstack.runtime import hooks
 from localstack.services.edge import ROUTER
 from localstack.services.s3.utils import S3_VIRTUAL_HOST_FORWARDED_HEADER
-from localstack.utils.aws.request_context import AWS_REGION_REGEX
 
 LOG = logging.getLogger(__name__)
 
+AWS_REGION_REGEX = r"(?:us-gov|us|ap|ca|cn|eu|sa)-[a-z]+-\d"
```

Comment:
Comment: While this regex is not as strict as the `AWS_REGION_REGEX` in the `request_context` module, it's definitely strict enough.